croquemort/crawler.py

```diff
@@ -11,6 +12,13 @@
 
 HEAD_TIMEOUT = 10  # in seconds
 GET_TIMEOUT = 3 * 60  # in seconds
+# domains that won't answer correctly to HEAD requests
+# TODO move to config file
+KNOWN_HEAD_OFFENDER_DOMAINS = [
+    'www.bnf.fr',
```

Comment:
I think it should not be declared within croquemort but udata-croquemort?

Reply:
Probably a good idea, cf https://github.com/opendatateam/croquemort/pull/42#issuecomment-328051895.